include/zephyr/drivers/sensor.h

```diff
@@ -356,10 +366,18 @@ enum sensor_attribute {
 
 	/** Hardware batch duration in ticks */
 	SENSOR_ATTR_BATCH_DURATION,
-	/* Configure the gain of a sensor. */
+	/** Configure the gain of a sensor. */
 	SENSOR_ATTR_GAIN,
-	/* Configure the resolution of a sensor. */
+	/** Configure the resolution of a sensor. */
 	SENSOR_ATTR_RESOLUTION,
+	/** Gender of the subject being monitored */
+	SENSOR_ATTR_GENDER,
```

Comment:
Feels kind of wrong to have this. Unless the sensor IC itself uses this information its something the application needs to do. A sensor does not have a gender.

Reply:
Hmm, have not seen that before. Interesting. Ok then 👍